src/main/java/at/hannibal2/skyhanni/features/garden/GardenOptimalSpeed.kt

```diff
@@ -84,9 +103,14 @@ class GardenOptimalSpeed {
 
         if (GardenAPI.hideExtraGuis()) return
 
-        val text = "Optimal Speed: §f$optimalSpeed"
+        var text = "Optimal Speed: §f$optimalSpeed"
         if (optimalSpeed != currentSpeed) {
+            text += " (§eCurrent: §f$currentSpeed"
+            if (sneaking) text += " §7[Sneaking]"
+            text += "§f)"
+
             if (config.showOnHUD) config.pos.renderString("§c$text", posLabel = "Garden Optimal Speed")
+            if (sneaking && !sneakingPersistent) return
```

Comment:
This logic means "no warning when sneaking for more than 5 seconds" and this cant be right. some loigcal error maybe?

Reply:
It means no warning if sneaking but not for long enough (5 seconds).

Reply:
ohh nvm